tests/test_get_plain_title.py

```diff
@@ -0,0 +1,145 @@
+import re
+
+SHOW_TITLE_REGEX = re.compile(r"^(?:(?:Episode)?\s?[0-9]+:+\s+)?(.+?)(?:(\s+\|+.*)|\s+)?$")
+
+
+def get_plain_title(title: str) -> str:
+    """
+    Get just the show title, without any numbering etc
+    """
+    return SHOW_TITLE_REGEX.match(title)[1]
```

Comment:
So, my only concern with this is that we're repeating ourselves with the regex. If we make a tweak in the scraper.py file then we'd also have to implement the same change here. This is what I'd recommend doing so we can ensure consistency in our tests and "application code".

```suggestion
from scraper import get_plain_title
```